string text
---
bool tts_ok